app/tooling/admin/main.go

```diff
@@ -53,6 +54,17 @@ func run() error {
 		return nil
 	}
 
+	//
+	if _, exists := flags["v"]; exists {
+		return commands.VaultInit(vault.Config{
```

Comment:
Let's talk about where/how we should actually be setting this as liar's admin works way differently than service admin does.